src/domain/user.js

```diff
@@ -19,15 +22,23 @@ export default class User {
       user.email,
       user.profile.bio,
       user.profile.githubUrl,
+      user.profile.profileUrl,
       user.password,
       user.role
     )
   }
 
   static async fromJson(json) {
     // eslint-disable-next-line camelcase
-    const { first_name, last_name, email, biography, github_url, password } =
-      json
+    const {
+      first_name,
```

Comment:
This is causing some linting errors. Before, the variables that we were creating were all being defined on one line. So we use this comment `// eslint-disable-next-line camelcase` to tell the linter to ignore the non-camel case variables.

We now need to add that comment above each of the variables in order to stop the linting from complaining that they're not using camel-case.